Blaise-Shyaka/Politico
# [#171721663] Create an API endpoint for admin to create office

# What does this PR do?

It creates an API endpoint allowing users to create an office

# Description of the tasks to be completed

- Write tests for the `POST /offices` endpoint
- Write a controller for that endpoint and mount it on a router
- Ensure that all the tests pass

# How should this be manually tested?

- Clone this repository
- Run `npm install` and `npm run test` to install dependencies and run tests
- All tests should pass

# What are the relevant pivotal tracker stories?

#171721663
